### ISSUE TITLE
Seems jquery not works in splash-jupyter

### ISSUE THREAD

**Issue by genius001** [scrapinghub/splash#417]
I'm not sure,

splash:autoload("https://code.jquery.com/jquery-2.1.3.min.js")

this works in HTTP UI, but not works in splash-jupyter.

**Comment by Youwotma** [comment]
True, seems like the autoload list gets cleared for every cell that is executed, if you load a page in the same cell as the autoload call it works, but in a different cell it doesn't

http://dev.scrapinghub.com/~david/autoload_jquery.html

**Comment by genius001** [comment]
I'm sorry. @Youwotma 

In my splash-jupyter:
```
splash:autoload("https://code.jquery.com/jquery-2.1.3.min.js")
assert(splash:go('http://dev.scrapinghub.com/~david/'))
splash:evaljs('"jQuery" in window')
```

always return false
